openml/_backport.py

```diff
@@ -0,0 +1,6 @@
+try:
```

Comment:
Since this import will only be used for testing, what do you think about moving it to the testing module?

Reply:
I was thinking about that. There's also some examples that could use it, but maybe there we can just do the ``try``.

Reply:
For the example I would always use the latest scikit-learn version if possible.